bin/test-lxd-images

```diff
@@ -82,6 +82,10 @@ lxc profile device add default root disk pool=default path=/
 # Create the instances
 echo "=> Creating the instances"
 for i in $(lxc image list images: | grep x86_64 | grep "${TYPE}" | grep "more" | awk '{print $2}'); do
+    # Skip Amazon Linux as it requires CGroupV1 which the testserver cannot satisfy
+    if [ "${i}" = "amazonlinux" ]; then
+        continue
+    fi
```

Comment:
This will also fail for centos/7, oracle/7, spingdalelinux/7 and ubuntu/16.04.

Instead of the hardcoded skip, it'd be better to look at:
```
stgraber@dakara:~$ lxc image show images:centos/7 | grep requirements.cgroup
  requirements.cgroup: v1
```